jx_sqlite/aggs_table.py

```diff
@@ -360,12 +360,21 @@ def _get(row):
                     column_number = len(outer_selects)
 
                     for json_type, sql in details.sql.items():
-                        concat_sql.append("GROUP_CONCAT(QUOTE(DISTINCT(" + sql + ")))")
+                        concat_sql.append("GROUP_CONCAT(DISTINCT(" + sql + "))")
```

Comment:
if you use the `json2value` suggestion I made, the union test passes.  Here is the diff I applied to your branch to make it work.

```diff
 jx_sqlite/aggs_table.py      | 3 ++-
 tests/test_jx/test_edge_1.py | 2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/jx_sqlite/aggs_table.py b/jx_sqlite/aggs_table.py
index b45f6b8..16c599b 100644
--- a/jx_sqlite/aggs_table.py
+++ b/jx_sqlite/aggs_table.py
@@ -14,6 +14,7 @@ from __future__ import division
 from __future__ import unicode_literals
 
 from mo_dots import listwrap, coalesce, split_field, join_field, startswith_field, relative_field, concat_field
+from mo_json import json2value
 from mo_logs import Log
 from mo_math import Math
 
@@ -399,7 +400,7 @@ class AggsTable(SetOpTable):
                         push_column_name=s.name,
                         push_column=si,
                         push_child=".",
-                        pull=sql_text_array_to_set(column_number),
+                        pull=lambda row: json2value(row[column_number]),
                         sql=concat_sql,
                         column_alias=_make_column_name(column_number),
                         type=sql_type_to_json_type[json_type]
diff --git a/tests/test_jx/test_edge_1.py b/tests/test_jx/test_edge_1.py
index 16ee31a..3b12512 100644
--- a/tests/test_jx/test_edge_1.py
+++ b/tests/test_jx/test_edge_1.py
@@ -558,7 +558,7 @@ class TestEdge1(BaseTestCase):
                 ]
             },
             "expecting_cube": {
-                "meta": {"format": "value"},
+                "meta": {"format": "cube"},
                 "data": {
                     "c": [2, 3, 4, 5, 6, 7, 8],
                     "a": ["x", "y"]
```